.github/workflows/build.yaml

```diff
@@ -70,9 +70,6 @@ jobs:
     runs-on: ubuntu-latest
     needs: [conda_build, waiting_room]
     if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')
-    defaults:
-      run:
-        shell: bash -el {0}
```

Comment:
This is not redundant! It contains `-l`, which means it runs a .bashrc, which is needed for conda to work.